libs/core/src/lib/static-utils/jwt-helper.ts

```diff
@@ -110,7 +114,7 @@ export class JwtHelper {
       return true
     } else {
       const date = this.getTokenExpirationDate(token)
-
+      // eslint-disable-next-line eqeqeq
```

Comment:
use `===` since `this.getTokenExpirationDate` will return `null` or a `Date`

Reply:
@michaelwittwer isn't it better to use `!date` instead?